src/Components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -32,15 +32,20 @@ const validateChildren = (children: React.ReactNode) => {
  * Creates a contextual menu by composing a MoreIcon (⋯) trigger
  * and a Dropdown component from Palette.
  */
-export const ContextualMenu: React.FC<ContextualMenuProps> = ({ children }) => {
+export const ContextualMenu: React.FC<ContextualMenuProps> = ({
+  children,
+  placement,
+  zIndex,
+}) => {
   validateChildren(children)
 
   return (
     <Dropdown
-      minWidth={["calc(100% - 40px)", 300]}
+      minWidth={200}
       openDropdownByClick
-      placement="bottom-end"
+      placement={placement}
       offset={0}
+      zIndex={zIndex}
```

Comment:
We need z-index + placement to be able to layer dropdowns but also have more flexibility when we use this component.